minions/prompts/minions.py

```diff
@@ -307,7 +307,11 @@ class JobOutput(BaseModel):
 ## Your Job: Write Two Python Functions
 
 ### FUNCTION #1: `prepare_jobs(context, prev_job_manifests, prev_job_outputs) -> List[JobManifest]`
-- Break the document(s) into chunks (using the provided chunking function, if needed). Determine the chunk size yourself according to the task: simple information extraction tasks can benefit from smaller chunks, while summarization tasks can benefit from larger chunks.
+- Break the document(s) into chunks using one of the provided chunking functions. Determine the document type and select the appropriate chunking function. Use information from the metadata and task to help you determine the most appropriate chunking function.
```

Comment:
Do you find that it can efficiently "determine the document type" without seeing it?
I wonder what is the best way for it to know.

@ANarayan what do you think about introducing an initial call where we ask the small model to provide info on the document it has and its contents? Minion (singular) style -- read the entire doc at batch_size=1